packages/core/src/external/carequality/ihe-gateway-v2/outbound/xca/process/dq-response.ts

```diff
@@ -77,9 +106,11 @@ function parseDocumentReference(
       (c: Classification) => c._classificationScheme === classificationScheme
     );
     if (!classification) return undefined;
-    const classificationSlots = Array.isArray(classification.Slot)
+
+    const slotArray = Array.isArray(classification.Slot)
```

Comment:
nit: I think we're doing this a lot throughout the code base. Maybe it's time we create a shared function that checks if the `unknown` type argument is an array or not and returns an array

Reply:
I'm gonna leave that for another time but good idea